src/scenes/motherScene.js

```diff
@@ -123,6 +123,18 @@ export class MotherScene extends Phaser.Scene {
         const obj_pos = new Phaser.Math.Vector2(cardObj.x, cardObj.y);
         cardObj.destroy();
 
+        if (cardObj.name === 'motherSceneBed_d') {
+            this.sound.add("clothes").play();
+        } else if (cardObj.name === 'motherSceneHamper_d') {
+            this.sound.add("cupboard").play();
+        } else if (cardObj.name === 'motherSceneBox_d' || cardObj.name === 'motherSceneXwing_d') {
+            this.sound.add("toys").play();
+        } else if (cardObj.name === 'motherSceneBooks_d' || cardObj.name === 'motherSceneColoring_d') {
+            this.sound.add("paper").play({volume: 0.5});
+        } else if (cardObj.name === 'motherSceneSwitch_d' || cardObj.name === 'motherSceneGlobe_d') {
+            this.sound.add("click").play({volume: 0.5});
+        }
+
```

Comment:
This should be done in each object's onclickCallback rather than here

Reply:
Okok, not sure I’ll have time this week but will try to do it soon

Reply:
Some objects share the same sound. IMHO it’s actually cleaner to control sounds here in `cleanObject()` rather than multiplying calls to "play", BUT: sounds should be added only once, I’ll change that

Reply:
(done in https://github.com/IMICovidou/Covidou/pull/147/commits/b1e121e7a70d3c165cea3bb466fe655eece4bb63)